src/openforms/forms/api/viewsets.py

```diff
@@ -608,6 +609,29 @@ def logic_rules_list(self, request, *args, **kwargs):
         )
         return Response(serializer.data, status=status.HTTP_200_OK)
 
+    @extend_schema(
+        summary=_("JSON schema"),
+        description=_("Generate the JSON schema for a form."),
+        parameters=[UUID_OR_SLUG_PARAMETER],
+        request=None,
+        responses={
+            (
+                status.HTTP_200_OK,
+                "application/json",
+            ): OpenApiTypes.OBJECT
+        },
+    )
+    @action(detail=True, methods=["get"], permission_classes=(permissions.IsAdminUser,))
+    def json_schema(self, request, *args, **kwargs):
+        form = self.get_object()
+
+        schema = generate_json_schema(
+            form,
+            limit_to_variables=form.formvariable_set.values_list("key", flat=True),
+        )
+
+        return JsonResponse(schema)
```

Comment:
please use `Response(...)` for this instead of `JsonResponse`, since this is a DRF endpoint.